src/pages/search.vue

```diff
@@ -95,6 +101,14 @@ const BrowsePage = {
         this.isAnyFilterApplied
       )
     },
+    searchFilter() {
+      return {
+        classes: this.isLargeScreen
+          ? 'column is-narrow grid-sidebar max-w-full bg-white'
+          : '',
```

Comment:
Can use the cleaner `{ 'list of classes': boolean }` notation here instead of ternary with blank string.